nextflow.config

```diff
@@ -43,6 +43,9 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
+    // FastMatch
+    threshold = 1.0
```

Comment:
This isn't a big thing, but I was thinking about how my output script is handling this and I was assuming it was an integer (hamming distances). We'll have to remember to accommodate both integers and floats with this.

Reply:
I think it's if scaled is provided it can be a float. Am I correct @apetkau?